internal/generator/templates/options.go.tpl

```diff
@@ -2,14 +2,17 @@
 package {{ .packageName }}
 
 import (
-    "github.com/pkg/errors"
+    "fmt"
     "golang.org/x/sync/errgroup"
 	"github.com/kazhuravlev/options-gen/pkg/validator"
 	goplvalidator "github.com/go-playground/validator/v10"
 	{{- range $import := .imports }}
 	{{ $import -}}
 	{{- end }}
 )
+{{ if .hasValidation }}
+var v = goplvalidator.New()
```

Comment:
Default behaviour for options-gen is generating `options_generated.go` near the `options.go`. That means that we can have a name collision.

Consider changing the name to something like `_optionsValidator` or something little-bit more unique.

Reply:
nice point
1) Rename to `_validator461e464ebed9`.
2) Open https://github.com/kazhuravlev/options-gen/issues/15.